submanifests/optional.yaml

```
@@ -67,7 +67,7 @@ manifest:
         - optional
     - name: zscilib
       path: modules/lib/zscilib
-      revision: ee1b287d9dd07208d2cc52284240ac25bb66eae3
+      revision: 9c715f135581dfd65df4ba63464391261f5b4583
```

Please update this to:

```suggestion
      revision: ee3c0c405087e331aad16d167b6e4ec1c3452ba9
```

---

done